### PR TITLE
closes #73 Insert Treatment 'did not happen' concepts into the OBSERVATION table.

### LINKED ISSUE #73: Insert Treatment 'did not happen' concepts into the OBSERVATION table.
The vocabulary work for this task was completed in #68.

### PR DESCRIPTION
**Pull request by mgurley**

### CODE REVIEW

File: etl/naaccr_etl_sqlserver.sql [hunk 1]
```diff
@@ -1334,9 +1330,186 @@ CREATE TABLE naaccr_data_points_temp
...
+      32527                             AS domain_concept_id_1			  -- Episode
+    , ep.episode_id                     AS fact_id_1
+    , 27                                AS domain_concept_id_2				-- Observation
+    , ob.observation_id                 AS fact_id_2
+    , 44818750                          AS relationship_concept_id  	-- Has occurrence
+    , NULL record_id
+   FROM episode_temp ep INNER JOIN observation_temp ob ON ep.person_id = ob.person_id AND ep.record_id = ob.record_id AND ep.episode_concept_id = 32528;			-- Disease First Occurrence
+
+
+  INSERT INTO fact_relationship
```
Comment (rtmill) [inline etl/naaccr_etl_sqlserver.sql]: If you run the ETL more than once, wouldn't the fact_relationship records from the previous runs still exist? If so we need to delete them beforehand.

File: etl/naaccr_etl_sqlserver.sql [hunk 2]
```diff
@@ -1334,9 +1330,186 @@ CREATE TABLE naaccr_data_points_temp
 	WHERE record_id        = det.rec_id
 	AND episode_concept_id = 32531; --Treatment Regimen
```
Comment (rtmill) [inline etl/naaccr_etl_sqlserver.sql]: We need a DELETE statement for the observation to data from previous runs. Example from above: 

```
DELETE FROM procedure_occurrence
WHERE procedure_type_concept_id = 32534;
```

File: etl/naaccr_etl_sqlserver.sql (hunk 1 above)
Comment (mgurley) [reply etl/naaccr_etl_sqlserver.sql]: Make sense.

File: etl/naaccr_etl_sqlserver.sql (hunk 2 above)
Comment (mgurley) [reply etl/naaccr_etl_sqlserver.sql]: Makes sense.

**Comment by mgurley** [comment]
This got done in https://github.com/OHDSI/OncologyWG/pull/246

**Pull request closed without merging**